sw/device/lib/aes.h

```diff
@@ -0,0 +1,95 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef _F_LIB_AES_H__
+#define _F_LIB_AES_H__
+
+#include <stddef.h>
+#include <stdint.h>
+
+/**
+ * Supported AES modes
+ */
+typedef enum aes_modes { AES_ENC = 0, AES_DEC = 1 } aes_modes_t;
+
+/**
+ * AES unit configuration options.
+ */
+typedef struct hmac_cfg {
```

Comment:
The name of the struct should match the name of the typedef.

Reply:
Good catch, thanks.